hasura/metadata/actions.graphql

```diff
@@ -1,44 +1,32 @@
 type Mutation {
-  adminUpdateUser(
-    object: AdminUpdateUserInput!
-  ): UserResponse
+  adminUpdateUser(payload: AdminUpdateUserInput!): UserResponse
 }
 
 type Mutation {
-  createUser(
-    object: CreateUserInput!
-  ): UserResponse
+  createUser(payload: CreateUserInput!): UserResponse
 }
 
 type Mutation {
-  create_circle(
-    object: create_circle_input!
-  ): create_circle_response
+  createCircle(payload: CreateCircleInput!): CreateCircleResponse
 }
 
 type Mutation {
-  logoutUser: logoutResponse
+  logoutUser: LogoutResponse
 }
 
 type Mutation {
-  upload_circle_logo(
-    object: upload_circle_image_input!
-  ): update_circle_response
+  uploadCircleLogo(payload: UploadCircleImageInput!): UpdateCircleResponse
 }
 
 type Mutation {
-  upload_profile_avatar(
-    object: upload_image_input!
-  ): update_profile_response
+  uploadProfileAvatar(payload: UploadImageInput!): UpdateProfileResponse
 }
 
 type Mutation {
-  upload_profile_background(
-    object: upload_image_input!
-  ): update_profile_response
+  uploadProfileBackground(payload: UploadImageInput!): UpdateProfileResponse
 }
 
-input create_circle_input {
+input CreateCircleInput {
   user_name: String!
   address: String!
```

Comment:
not sure how the `address` came back, we removed it during the address spoofing fix